apps/web/src/components/workflow/ChannelNode.tsx

```diff
@@ -9,7 +9,7 @@ interface NodeData {
   tabKey: string;
   index: number;
   testId: string;
-  templateId: string;
```

Comment:
Look like you removed the templateId that you added on https://github.com/novuhq/novu/pull/660 it was not needed in the end?